src/main/resources/edu/cornell/kfs/fp/document/validation/configuration/FinancialProcessingValidators.xml

```diff
@@ -25,7 +25,7 @@
 	<bean id="DisbursementVoucher-employeeInformationValidation" class="edu.cornell.kfs.fp.document.validation.impl.CuDisbursementVoucherEmployeeInformationValidation" abstract="true" />
```

Comment:
This bean now needs to have its services injected. Some of the other beans in this file may need to be updated likewise.

Reply:
Added service injection for beans in FinancialProcessingValidation.xml